projectfiles/Xcode/Wesnoth.xcodeproj/project.pbxproj

```diff
@@ -6175,21 +6241,12 @@
 				);
 				INFOPLIST_FILE = Info.plist;
 				INSTALL_PATH = "$(HOME)/Applications";
-				LIBRARY_SEARCH_PATHS = (
-					"$(inherited)",
-					"\"$(SRCROOT)/lib\"",
-				);
 				OTHER_CPLUSPLUSFLAGS = (
 					"$(OTHER_CFLAGS)",
 					"-include",
 					"../../src/boost-patched/bind/arg.hpp",
 					"-I/usr/local/opt/openssl@1.1/include",
 				);
-				"OTHER_CPLUSPLUSFLAGS[arch=*]" = (
```

Comment:
This too looks suspicious; see my previous comment.

Reply:
This is override of override for specific architecture, which is undefined so every (`*`) and doesn't include `"-I/usr/local/opt/openssl@1.1/include",` . Normal override is on line 6244. This one is useless.

Reply:
I think what he's referring to (or, at least, what makes me suspicious) is deleting a left-parenthesis `(` without also deleting a right `)`

Reply:
You must look at full change. @CelticMinstrel commented just first line of change, but it is longer:
 
<img width="1041" alt="image" src="https://user-images.githubusercontent.com/2218817/35942572-649cab16-0c56-11e8-9528-0350ed2c86ee.png">

Reply:
Ah ok

Reply:
Well it should, this remained from someone and I forgot change it, I'll change it now.

Reply:
Fixed by https://github.com/wesnoth/wesnoth/pull/2450/commits/721f6d88f71c695db0f1072050bcc753cb6dff78